spec/spec_helper.rb

```diff
@@ -87,12 +72,21 @@
 
   config.include RakeTaskExampleGroup, :type => :rake_task
 
+
+  # config.before(:all) do
+  #   EvmSpecHelper.log_ruby_object_usage
+  # end
+  # config.after(:all) do
+  #   EvmSpecHelper.log_ruby_object_usage
+  # end
```

Comment:
These looked useful, so I just moved them down by the other callbacks where they belong.